lms/djangoapps/support/static/support/jsx/logged_in_user.jsx

```diff
@@ -34,10 +34,10 @@ function LoggedInUser({ userInformation, setErrorState, zendeskApiHost, submitFo
     </div>);
   }
 
-  let subjectElement;
-  subjectElement = (<div>
-    <label htmlFor="subject">{gettext('Subject')}</label>
-    <select className="form-control select-subject" id="subject">
+  let topicElement;
+  topicElement = (<div>
+    <label htmlFor="topic">{gettext('Topic')}</label>
+    <select className="form-control select-subject" id="topic">
```

Comment:
nit: is the css class `select-subject` used in other places?  if so i would say its fine to just leave it be.  if its only used here though, maybe we want to update it to say `select-topic` just so its stays consistent with all the other changes?